lib/facter/self_service.rb

```diff
@@ -112,4 +112,17 @@
     # Is there at least 9% memory available
     { S0021: Facter.value(:memory)['system']['capacity'].to_f <= 90 }
   end
+  
+  chunk(:S0033) do
+    hiera_config_path = Puppet.settings['hiera_config']
+    next unless File.exist?(hiera_config_path)
+    # Is Hiera 5 in use?
+    hiera_config_file = YAML.load_file(hiera_config_path)
+    if hiera_config_file.dig('version') == 5
```

Comment:
```suggestion
 { S0033: hiera_config_file.dig('version') == 5 }
```

The IF statement can be condensed into a single line, as in ruby the default output of a any expression is boolean

Reply:
Didn't realize dig would not fail if the key doesn't exist. Fixed this!